yield_plugin/src/yield_plugin.cpp

```diff
@@ -135,17 +135,17 @@ namespace yield_plugin
     if (incoming_request.strategy == "carma/cooperative-lane-change")
     {
       if (!map_projector_) {
-        RCLCPP_ERROR_STREAM(nh_->get_logger(),"Cannot process mobility request as map projection is not yet set!");
+        RCLCPP_ERROR_STREAM(rclcpp::get_logger("yield_plugin"),"Cannot process mobility request as map projection is not yet set!");
```

Comment:
Is there a reason this was changed? Using the `rclcpp::` version tightly couples logging to the Node name. If a user were to change the Node's name, we would suddenly lose logging.

Also, we don't need the `_STREAM` logging variant here, we should use `RCLCPP_ERROR`.

Reply:
This was because there was a bug in the way log levels set in carma-config. See `https://github.com/usdot-fhwa-stol/carma-utils/pull/151`

Reply:
Since it is working now, I changed it to use node's logger